README.md

```diff
@@ -196,6 +196,51 @@ aws_start_me_up = Rouster.new(
 )
 
 ```
+### instantiation for openstack
```

Comment:
i'd rather that this was added to the same block as the AWS examples (advanced instantiation), and followed the same form. adding the example (and renaming to follow the convention set out already, that is `openstack.rb`) to line 152 would allow you to provide this same information